bin/start-chronos.bash

```diff
@@ -1,27 +1,39 @@
 #!/bin/bash
-set -o errexit -o nounset -o pipefail
+set -o errexit -o pipefail
 
-chronos_home="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd -P )"
-echo "Chronos home set to $chronos_home"
-export JAVA_LIBRARY_PATH="/usr/local/lib:/lib:/usr/lib"
-export LD_LIBRARY_PATH="${LD_LIBRARY_PATH:-/lib}"
-export LD_LIBRARY_PATH="$JAVA_LIBRARY_PATH:$LD_LIBRARY_PATH"
```

Comment:
@eling `[ -z "$1"]` should work just fine when first arg is not present. That's why it is wrapped in double-quotes. `-z` is true when STRING length is zero, so when `$1` is not passed, bash would resolve it to empty string and as long as it's quoted, we are good. Unless of course, we have `nounset` set :). Arg 0 always exists.. though we don't really care.